include/items.php

```diff
@@ -1720,11 +1721,7 @@ function new_follower($importer, $contact, $datarray, $item, $sharing = false) {
 							'hash' => $hash, 'datetime' => datetime_convert()));
 			}
 
-			$def_gid = get_default_group($importer['uid'], $contact_record["network"]);
-
-			if (intval($def_gid)) {
-				group_add_member($importer['uid'], '', $contact_record['id'], $def_gid);
-			}
+			Group::addMember(User::getDefaultGroup($importer['uid'], $contact_record["network"]), $contact_record['id']);
```

Comment:
Before the change there was this ```if``` that checked whether the default group was set. Is this covered in ```Group::addMember``` now?

Reply:
Yep, if the group id is invalid it doeasn’t add anything.